features/ASM000_Assemblies-information.feature

```diff
@@ -0,0 +1,21 @@
+@implementer-agreement
+@ASM
+@version1
+@E00020
+
+Feature: ASM000 - Assemblies information
+    The rule verifies the presence of IFC entities used to model elements composed of / constructed by other elements.
+    For example, a roof might be assembled from a series of prefabricated truss components.
+    https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/concepts/Object_Composition/Aggregation/Element_Composition/content.html
+
+
+    Scenario: Check for activation
+
+    Given an IfcElementAssembly
```

Comment:
We have to take a second here to think about what this functional part means.

- Does it mean the usage of IfcElementAssembly
- Or does it mean the usage of the IfcRelAggregates relationship, which can equally well be applied to e.g a wall or a stair.

(I would actually say the 2nd one, because IfcElementAssembly is a bit the proxy element of compositions.

Note from the docs:

> The [IfcElementAssembly](https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcElementAssembly.htm) is a general purpose entity that is required to be decomposed. **Also other subtypes of [IfcElement](https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcElement.htm) can be decomposed**

(emphasis mine)

Reply:
Fair enough. If what this FP entails is the ability of applications to define assemblies (ie, elements made of other elements) then I'd say it's ok to have it generic (ie, Given an IfcElement or its subtypes)

Reply:
[Reversed it to IfcElement](https://github.com/buildingSMART/ifc-gherkin-rules/pull/255/commits/6056105fa6493832381c41a50b742a8c0c54448e)